modules/configuration-annotation-processor/src/integrationTest/java/org/apache/ignite/internal/configuration/processor/ItProcessorTest.java

```diff
@@ -287,21 +409,25 @@ void testStaticConstants() {
      */
     private static BatchCompilation batchCompile(String packageName, String... classNames) {
         ClassName[] classes = Arrays.stream(classNames)
-                .map(clsName -> ClassName.get(packageName, clsName))
-                .toArray(ClassName[]::new);
+            .map(clsName -> ClassName.get(packageName, clsName))
+            .toArray(ClassName[]::new);
 
         return batchCompile(classes);
     }
 
-    @Test
-    void wrongSchemaPostfix() {
-        String packageName = "org.apache.ignite.internal.configuration.processor";
-
-        ClassName schema = ClassName.get(packageName, "ConfigurationSchemaWithWrongPostfix");
-
-        Compilation compilation = compile(schema);
+    /**
+     * Extends {@link Assertions#assertThrows(Class, Executable)} to check for a substring in the error message.
+     *
+     * @param expErrCls Expected error class.
+     * @param exec Supplier.
+     * @param expSubStr Expected substring in error message.
+     * @throws AssertionFailedError If failed.
+     */
+    private void assertThrowsEx(Class<? extends Throwable> expErrCls, Executable exec, @Nullable String expSubStr) {
+        Throwable t = assertThrows(expErrCls, exec);
 
-        assertThat(compilation).failed();
-        assertThat(compilation).hadErrorContaining(schema + " must end with 'ConfigurationSchema'");
+        if (expSubStr != null) {
+            assertTrue(t.getMessage().contains(expSubStr), () -> String.format("%s not contains %s", t.getMessage(), expSubStr));
```

Comment:
There's a org.hamcrest.Matchers#containsString that will save you from creating error message manually